### PR TITLE
initial draft of sequencing and variant calling

### PR DESCRIPTION
**Pull request by bdo311**
Thanks in advance for the feedback! I'll probably try to revise some of the wording to improve the flow.

### CODE REVIEW

File: sections/04_study.md [hunk 1]
```diff
@@ -605,5 +605,65 @@ unlabeled (microbiome experiments) examples.
...
+obtain the genomic data itself. We will discuss two specific challenges: calling
+SNPs (single nucleotide polymorphisms) and indels (insertions and deletions)
+with high specificity and sensitivity, and improving the accuracy of new types
+of data such as nanopore sequencing. These two tasks are critical for studying
+rare variation, allele-specific transcription and translation, and splice site
+mutations, among others. In the clinical realm, sequencing of rare tumor clones
+and other genetic diseases will require accurate calling of SNP and indels.
+
+Current methods achieve relatively high (>99%) precision at 90% recall for SNPs
+and indel calls from Illumina short-read data, yet this leaves a large number of
```
Comment (agitter) [inline sections/04_study.md]: Do we have a reference we can use for this performance level?

File: sections/04_study.md [hunk 2]
```diff
@@ -605,5 +605,65 @@ unlabeled (microbiome experiments) examples.
...
+encodes the base (A, C, T, G) as a different R value, quality score as a G
+value, strand as a B value, and variation from the reference as the alpha value.
+The neural network outputs genotype probabilities for each candidate variant.
+They were able to achieve better performance than GATK, a leading genotype
+caller, even when GATK was given information about population variation for each
+candidate variant. Another method, still in its infancy, hand-developed 642
+features for each candidate variant and fed these vectors into a fully connected
+deep neural network [@tag:Torracinta2016_deep_snp]. Unfortunately, this feature
+set required at least 15 iterations of software development to fine-tune, which
+will likely not be generalizable. Going forward, we foresee that variant calling
```
Comment (agitter) [inline sections/04_study.md]: Great conclusions, I'm always happy to see us take a stance in the review.  To challenge you slightly, what do you think about the tension between natural encodings of the data where domain-specific labeled data may be limited versus conceptually suboptimal encodings (e.g. DNA -> images) where labeled data are rich or pre-trained models are available?  Is this image-to-variant calling strategy likely to hold an advantage only in the short term until better training datasets or simulators become available?  Should it be more widely adopted elsewhere in genomics?

File: sections/04_study.md [hunk 3]
```diff
@@ -605,5 +605,65 @@ unlabeled (microbiome experiments) examples.
...
+will benefit most from optimized neural network architectures and better methods
+of encoding raw sequence and quality data (e.g. in tensor rather than RGBA
+format).
+
+In limited experiments, DeepVariant was robust to sequencing depth, read length,
+and even species [@tag:Poplin2016_deepvariant]. However, a model built on
+Illumina data, for instance, will likely not be applicable to PacBio long-read
+data or MinION nanopore data, which have vastly different specificity and
+sensitivity profiles and signal-to-noise characteristics. Recently, Boza et al.
+used bidirectional recurrent neural networks infer the E. coli sequence from
```
Comment (agitter) [inline sections/04_study.md]: _E. coli_ in italics

File: sections/04_study.md (hunk 1 above)
Comment (bdo311) [inline sections/04_study.md]: Yep I'll re-reference the Poplin paper, which benchmarked the previous state of the art algorithms (GATK).

File: sections/04_study.md (hunk 2 above)
Comment (bdo311) [inline sections/04_study.md]: I think it's an issue of not really knowing the best representation with which to input your data. With sequence data, we could either encode it as an RGB image and send it through a CNN already pretrained on ImageNet, or encode it as a one-hot tensor and send it through a CNN that we train and optimize explicit for genomic data. To know which one is better will require more work to be done. 

I think what I was trying to get across in the section was that inputting raw sequence (in its optimal representation) could probably in the long run be superior or equal to inputting sequence + hand-derived features (in its optimal representation) in terms of time spent optimizing.

Comment (agitter) [inline sections/04_study.md]: The point about raw sequence versus hand-derived features came across clearly.  If we think comparing image-based versus natural encodings will require more work to be done, I'll merge this as-is.

Comment (bdo311) [inline sections/04_study.md]: I can try to clarify the point about image based vs natural encodings in a sentence -- just to say that we don't know which will be best, just that there needs to be more research to test both strategies out under otherwise identical conditions.

Comment (agitter) [inline sections/04_study.md]: Okay.  You only need to add that if you really want to.  I merged already so you would need to open another small pull request.
